ksp_plugin/orbit_analyser.cpp

```diff
@@ -182,10 +180,12 @@ Status OrbitAnalyser::RepeatedlyAnalyseOrbit() {
     {
       absl::MutexLock l(&lock_);
       next_analysis_ = std::move(analysis);
+      // Take ownership of our own thread so we can safely detach.
+      analyser = std::move(analyser_);
```

Comment:
FWIW, the usual pattern to do this is to `swap`.  This avoids having to reason about moved-from objects (`analyzer_` becomes a well-defined, default-initialized `jthread`.